saar

```diff
@@ -10,6 +10,11 @@ networks:
   ipv6:
     - fd4e:f2d7:88d2:ffff::/64
     - fd4e:f2d7:88d2:fffd::/64
+    - fd4e:f2d7:88d2:fffc::/64
+    - fd4e:f2d7:88d2:fffb::/64
+    - fd4e:f2d7:88d2:fffa::/64
+    - fd4e:f2d7:88d2:fff9::/64
+    - fd4e:f2d7:88d2:fff8::/64
```

Comment:
Maybe instead of allocating multiple `/64` take a shorter prefix.

Reply:
Unfortunately, `fd4e:f2d7:88d2:fffe` is already used by the luxembourg community.